biohansel/utils.py

```diff
@@ -76,43 +80,16 @@ def find_inconsistent_subtypes(subtypes: List[List[int]]) -> List[str]:
     return incon_subtypes
 
 
-def get_scheme_fasta(scheme: str) -> str:
-    if scheme in SCHEME_FASTAS:
-        scheme_fasta = SCHEME_FASTAS[scheme]['file']
-    elif os.path.exists(scheme) and os.path.isfile(scheme):
-        scheme_fasta = scheme
-    else:
-        raise FileNotFoundError('Could not find user-specified subtyping scheme fasta "%s"', scheme)
-    return scheme_fasta
-
-
-def get_scheme_params(scheme: str) -> Optional[SubtypingParams]:
-    if scheme in SCHEME_FASTAS:
-        return SCHEME_FASTAS[scheme]['subtyping_params']
-
-
-def get_scheme_version(scheme: str) -> Optional[str]:
-    if scheme in SCHEME_FASTAS:
-        version = SCHEME_FASTAS[scheme]['version']  # type: str
-        return version
-    return None
-
-
 def collect_fastq_from_dir(input_directory: str) -> List[Union[str, Tuple[List[str], str]]]:
     fastqs = []
     for x in os.listdir(input_directory):
```

Comment:
just a small thing, but perhaps a more descriptive variable name for x?

Reply:
Changed in 51dafd0c15011c1e027d93b4a9ff67ead30df447